Classes/CalcBreakdownControl.lua

```diff
@@ -537,14 +537,12 @@ function CalcBreakdownClass:DrawBreakdownTable(viewPort, x, y, section)
 				local _, alpha = string.gsub(row[col.key], "%a", " ") -- counts letters in the string
 				local _, notes = string.gsub(row[col.key], " to ", " ") -- counts " to " in the string
 				local _, paren = string.gsub(row[col.key], "%b()", " ") -- counts parenthesis in the string
-				if main.showThousandsCalcs and (alpha == 0 or notes > 0 or paren > 0) and col.right then
+				if (alpha == 0 or notes > 0 or paren > 0) and col.right then
 					DrawString(col.x + col.width - 4, rowY + 1, "RIGHT_X", 12, "VAR", "^7"..formatNumSep(tostring(row[col.key])))
-				elseif col.right then
-					DrawString(col.x + col.width - 4, rowY + 1, "RIGHT_X", 12, "VAR", "^7"..row[col.key])
```

Comment:
Was this condition meant to be dropped?

Reply:
The conditional is checked inside formatNumSep.